repository/src/main/java/uk/gov/hmcts/ccd/definition/store/repository/model/AccessTypeRolesResult.java

```diff
@@ -0,0 +1,111 @@
+package uk.gov.hmcts.ccd.definition.store.repository.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+import java.util.List;
+
+@ApiModel(description = "accessType")
+public class AccessTypeRolesResult {
+
+    private String organisationProfileId;
+    private String accessTypeId;
+    private Boolean accessMandatory;
+    private Boolean accessDefault;
+    private Boolean display;
+    private String description;
+    private String hint;
+    private Integer displayOrder;
+
+    private List<AccessTypeRolesRoleResult> roles;
+
+    @ApiModelProperty(required = true, value = "")
+    @JsonProperty("organisationProfileId")
+    public String getOrganisationProfileId() {
+        return organisationProfileId;
+    }
+
+    public void setOrganisationProfileId(String organisationProfileId) {
+        this.organisationProfileId = organisationProfileId;
+    }
+
+    @ApiModelProperty(required = true, value = "")
```

Comment:
not sure about the any advantage f having these ApiModelProperty with blank values - does it need any description?